apis/v1alpha1/httproute_types.go

```diff
@@ -167,6 +167,18 @@ type HTTPRouteRule struct {
 	// path match on "/", which has the effect of matching every
 	// HTTP request.
 	//
+	//
+	// A client request may match multiple HTTP route rules,
+	// to differentiate we define matching precedence where matches
+	// MUST be determined in order of the following criteria, continuing on ties:
+	// * longest match hostname
+	// * longest match path
+	// * largest number of header matches
```

Comment:
The list items above all start with lowercase characters. The list items below all start with uppercase characters, specifically `The` and end with `.`. I think I prefer the style below, maybe update the above to match that?